test/StringDiagram.lean

```diff
@@ -57,4 +59,30 @@ example {X₁ Y₁ X₂ Y₂ : C} (f : X₁ ⟶ Y₁) (g : X₂ ⟶ Y₂) : f 
   rw [MonoidalCategory.whisker_exchange]
   rw [MonoidalCategory.tensorHom_def]
 
+end
+
+variable {C : Type u} [Category.{v} C] [i : MonoidalCategory C] {X Y : C}
+
+#string_diagram MonoidalCategory.whisker_exchange
+#string_diagram MonoidalCategory.whisker_exchange_assoc
+#string_diagram MonoidalCategory.pentagon
+#string_diagram MonoidalCategory.whiskerLeft_id
+#string_diagram left_triangle
+#string_diagram yang_baxter
+#string_diagram yang_baxter'
+#string_diagram yang_baxter''
+#string_diagram Category.assoc
+#string_diagram Functor.map_comp
+#string_diagram NatTrans.naturality
```

Comment:
@yuma-mizuno @Vtec234 at the moment these tests might start "failing" without us noticing, because we don't record expected output. Can we adapt the tests somehow? Maybe use `#guard_msg` on a slightly more low-level command?

Reply:
I think it might be possible and reasonable to check the [penrose substance programs](https://penrose.cs.cmu.edu/docs/ref/substance/overview) the string diagram widget creates is what we expect. I'll try to write some test code like that.